Dockerfile

```diff
@@ -66,5 +66,8 @@ RUN cp ${WEB_REPO}/config.dist.php ${WEB_REPO}/config.php && \
     -e "s/\['port'\] = 3306;/\['port'\] = 3306;\n\n\$password_file = getenv(\"MYSQL_ENV_MYSQL_PASSWORD_FILE\");\nif(file_exists(\$password_file))\n\$db\['pass'\] = preg_replace(\"\/\\\\s+\/\", \"\", file_get_contents(\$password_file));/" \
     ${WEB_REPO}/config.php
 
+#set upload folder permission
+RUN chmod 777 /var/www/html/app/subnets/import-subnet/upload
```

Comment:
As you can see just above https://github.com/pierrecdn/phpipam/pull/29/files#diff-3254677a7917c6c01f55212f86c57fbfR61, the rights are already given to `www-data` (Apache httpd) in this directory.
Why would you need to allow 0777 perms here? Something else that Apache has to be able to read/write?

Reply:
Also, you don't want to create an additional container layer just for that, you have to embed into the previous RUN statement.

Reply:
To Upload Ip Adresses with a .csv file the upload folder needs 777 permissions (as descriped in the link above), otherwhise there will be the following error: "Cannot move file to upload dir".